src/main/java/com/griefprevention/commands/ClaimCommand.java

```diff
@@ -71,8 +71,7 @@ public boolean onCommand(
             try
             {
                 radius = Integer.parseInt(args[0]);
-            }
-            catch (NumberFormatException e)
+            } catch (NumberFormatException e)
```

Comment:
Is this how GP wants to handle catches? I thought they were usually placed on separate lines from the closing braces, but I could be wrong.